<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotWindow</class>
 <widget class="QMainWindow" name="PlotWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plots - qAgate</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="saveLeft">
         <property name="toolTip">
          <string>Save left graph</string>
         </property>
         <property name="statusTip">
          <string>Save left graph</string>
         </property>
         <property name="whatsThis">
          <string>Save left graph</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="restoreLeft">
         <property name="toolTip">
          <string>Restore left view</string>
         </property>
         <property name="statusTip">
          <string>Restore left view</string>
         </property>
         <property name="whatsThis">
          <string>Restore left view</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="restoreRight">
         <property name="toolTip">
          <string>Restore right view</string>
         </property>
         <property name="statusTip">
          <string>Restore right view</string>
         </property>
         <property name="whatsThis">
          <string>Restore right view</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="saveRight">
         <property name="toolTip">
          <string>Save right graph</string>
         </property>
         <property name="statusTip">
          <string>Save right graph</string>
         </property>
         <property name="whatsThis">
          <string>Save right graph</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QTabWidget" name="left">
       <property name="tabPosition">
        <enum>QTabWidget::West</enum>
       </property>
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QTabWidget" name="right">
       <property name="tabPosition">
        <enum>QTabWidget::East</enum>
       </property>
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSaveLeft">
   <property name="icon">
    <iconset resource="../qagate/qagate.qrc">
     <normaloff>:/MD/icons/save.svg</normaloff>:/MD/icons/save.svg</iconset>
   </property>
   <property name="text">
    <string>SaveLeft</string>
   </property>
   <property name="toolTip">
    <string>Save left graph</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveRight">
   <property name="icon">
    <iconset resource="../qagate/qagate.qrc">
     <normaloff>:/MD/icons/save.svg</normaloff>:/MD/icons/save.svg</iconset>
   </property>
   <property name="text">
    <string>SaveRight</string>
   </property>
   <property name="toolTip">
    <string>Save right graph</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionRestoreLeft">
   <property name="icon">
    <iconset resource="../qagate/qagate.qrc">
     <normaloff>:/MD/icons/zoom-1.svg</normaloff>:/MD/icons/zoom-1.svg</iconset>
   </property>
   <property name="text">
    <string>RestoreLeft</string>
   </property>
   <property name="toolTip">
    <string>Restore left view</string>
   </property>
   <property name="shortcut">
    <string>U</string>
   </property>
  </action>
  <action name="actionRestoreRight">
   <property name="icon">
    <iconset resource="../qagate/qagate.qrc">
     <normaloff>:/MD/icons/zoom-1.svg</normaloff>:/MD/icons/zoom-1.svg</iconset>
   </property>
   <property name="text">
    <string>RestoreRight</string>
   </property>
   <property name="toolTip">
    <string>Restore right view</string>
   </property>
   <property name="shortcut">
    <string>Shift+U</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../qagate/qagate.qrc"/>
 </resources>
 <connections/>
</ui>
